### PR TITLE
fix: globalize messages from artifact generator

### PR DESCRIPTION
**Pull request by agnes512**
Fixes https://github.com/strongloop/loopback-next/issues/5131

Globalizes `xxx will be created in...` and `xxx was created in....` strings 
and other strings.

## Checklist

👉 [Read and sign the CLA (Contributor License Agreement)](https://cla.strongloop.com/agreements/strongloop/loopback-next) 👈

- [ ] `npm test` passes on your machine
- [ ] New tests added or existing tests modified to cover all changes
- [ ] Code conforms with the [style guide](http://loopback.io/doc/en/contrib/style-guide.html)
- [ ] API Documentation in code was updated
- [ ] Documentation in [/docs/site](../tree/master/docs/site) was updated
- [ ] Affected artifact templates in `packages/cli` were updated
- [ ] Affected example projects in `examples/*` were updated

👉 [Check out how to submit a PR](https://loopback.io/doc/en/lb4/submitting_a_pr.html) 👈

### CODE REVIEW

File: packages/cli/lib/utils.js [hunk 1]
```diff
@@ -114,7 +121,7 @@ exports.logNamingIssues = function (name, log) {
   if (name.match(/[\u00C0-\u024F\u1E00-\u1EFF]/)) {
     log(
       chalk.red('>>> ') +
-        `Accented chars in the class name will get replaced: ${name}`,
+        g.f('Accented chars in the class name will get replaced: ') + `${name}`,
```
Comment (dhmlau) [inline packages/cli/lib/utils.js:124]: I think we can include the name as a parameter, i.e. 
```
g.f('Accented chars in the class name will get replaced: {0}', `${name}`) 
```

File: packages/cli/lib/version-helper.js [hunk 2]
```diff
@@ -102,13 +103,19 @@ async function checkDependencies(generator) {
   const originalCliVersion = generator.config.get('update') || '<unknown>';
   generator.log(
     chalk.red(
-      `The project was originally generated by @loopback/cli@${originalCliVersion}.`,
+      g.f(
```
Comment (agnes512) [inline packages/cli/lib/version-helper.js:106]: image

File: packages/cli/lib/artifact-generator.js [hunk 3]
```diff
@@ -81,7 +81,17 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
    * >> Model MyModel will be created in src/models/my-model.model.ts
    **/
   promptClassFileName(type, typePlural, name) {
-    utils.logClassCreation(type, typePlural, name, this.log.bind(this));
+    this.log(
```
Comment (agnes512) [inline packages/cli/lib/artifact-generator.js:84]: image

File: packages/cli/lib/artifact-generator.js [hunk 4]
```diff
@@ -118,7 +128,7 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
       this.log(
         utils.toClassName(this.artifactInfo.type),
         chalk.yellow(classesOutput),
-        classes.length > 1 ? 'were created in' : 'was created in',
+        classes.length > 1 ? g.f('were created in') : g.f('was created in'),
```
Comment (agnes512) [inline packages/cli/lib/artifact-generator.js:131]: 截屏2020-04-21下午4 19 48

File: packages/cli/generators/relation/index.js [hunk 5]
```diff
@@ -498,11 +498,11 @@ module.exports = class RelationGenerator extends ArtifactGenerator {
       {
         type: 'confirm',
         name: 'registerInclusionResolver',
-        message: `Allow ${chalk.yellow(
-          this.artifactInfo.sourceModel,
-        )} queries to include data from related ${chalk.yellow(
-          this.artifactInfo.destinationModel,
-        )} instances?`,
+        message: g.f(
+          'Allow %s queries to include data from related %s instances? ',
```
Comment (agnes512) [inline packages/cli/generators/relation/index.js:502]: image

File: packages/cli/generators/discover/index.js [hunk 6]
```diff
@@ -233,7 +233,9 @@ module.exports = class DiscoveryGenerator extends ArtifactGenerator {
       if (props.disableCamelCase) {
         this.log(
           chalk.red(
-            'By disabling Camel case, you might need to specify these customized names in relation definition.',
+            g.f(
+              'By disabling Camel case, you might need to specify these customized names in relation definition.',
```
Comment (agnes512) [inline packages/cli/generators/discover/index.js:237]: 截屏2020-04-21下午5 18 19

Comment (nabdelgadir) [reply packages/cli/generators/discover/index.js:237]: Is "By disabling Camel case" supposed to still be in English?

Comment (agnes512) [reply packages/cli/generators/discover/index.js:237]: In the PR, they are supposed to be in English. And those screenshots are showing that  the globalized string in English can be translated ( but we don't do the translation ourselves).

**Pull request merged**